bin/templates/regress_macros.j2

```diff
@@ -1,3 +1,3 @@
 {%- macro yesorno(val) -%}
-{%- if val -%}YES{%- else -%}NO{%- endif -%}
+{%- if val == '0' -%}NO{%- elif val == '1' -%}YES{%- else -%}{{val}}{%- endif -%}
```

Comment:
This simply supports setting 1 or 0 in lieu of yes or no from the cv_regress command line.  Since our contributors seem to be evenly split amongst preferring YES or 1 (and I prefer 0 or 1) we support both.